packages/Makefile

```diff
@@ -188,20 +188,20 @@ test_bridge.android: build_core.android
 	fi;	\
 	emulator -avd $$EMULATOR -no-boot-anim -no-window -no-snapshot-save -gpu swiftshader_indirect -noaudio & EMULATOR_PID=$$!; \
 	adb wait-for-device shell 'while [[ -z $$(getprop sys.boot_completed) ]]; do sleep 1; done;'; \
-	(cd $(ANDROID_DIR) && ./gradlew bridge:connectedAndroidTest); \
-	kill $$EMULATOR_PID
-	@echo 'Done!'
+	(cd $(ANDROID_DIR) && ./gradlew bridge:connectedAndroidTest) || \
+	(kill $$EMULATOR_PID; exit 1) && \
```

Comment:
This line makes me think that everything after will never run

Reply:
You made me doubt ^^ but I've just retested the Makefile on both bash and zsh and it works fine (I admit that the bash syntax isn't the most intuitive in the world).